README.md

```diff
@@ -92,9 +92,8 @@ I converted this simple, one-cmdlet script to a module and published it so anyon
 | slmgr /del                            | not implemented                                                                 | |
 | slmgr /atp                            | not implemented                                                                 | |
 | slmgr /skms-domain                    | not implemented                                                                 | |
-| slmgr /ckms                           | not implemented                                                                 | |
-| slmgr /ckms                           | not implemented                                                                 | |
-| slmgr /cpky                           | not implemented                                                                 | |
+| slmgr /ckms                           | https://github.com/zbalkan/slmgr-ps/issues/8#issue-1947051438                   | |
+| slmgr /cpky                           | https://github.com/zbalkan/slmgr-ps/issues/8#issue-1947051438                   | |
 | slmgr /dti                            | not implemented                                                                 | |
 | slmgr /ilc                            | not implemented                                                                 | |
 | slmgr /rilc                           | not implemented                                                                 | |
```

Comment:
Shouldn't we create issues for all of these so that potential users (like myself) can track them? If they're notified when each feature is implemented, they'll be able to use this module the moment it's useful, rather than having to periodically check.

Reply:
I'd suggest making a roadmap of which things you plan to do so that if the community wants to see X or Y capability, they know whether to wait on you or roll up their sleeves and start coding.